packages/cli/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "module": "commonjs",
```

Comment:
Nx generated it like this by default, but its actually ignored in the `build` because we're using `esbuild` instead of `tsc`.

But I think I'll change these to `ESNext` anyway, otherwise I think TypeScript would complain if we used some ESM-only syntax (like `import.meta.url`).